src/plugins/fs.c

```diff
@@ -373,6 +491,33 @@ static gboolean wipe_fs (const gchar *device, const gchar *fs_type, GError **err
         return FALSE;
     }
 
+    if (wipe_all) {
+        gboolean has_fat = TRUE;
+        has_fat = has_fs (device, "vfat", error);
```

Comment:
This cannot be vfat-specific, it needs to take the ``fs_type`` parameter into account.